packages/math-input/src/components/tabbar/__tests__/item.test.tsx

```diff
@@ -57,7 +57,7 @@ describe("<TabbarItem />", () => {
                 onClick={() => {}}
             />,
         );
-        jest.runAllTimers();
+        act(() => jest.runOnlyPendingTimers());
```

Comment:
React 18/RTL are much more sensitive to element updates. Most of the time when we call `runOnlyPendingTimers()` in Perseus tests it's because we're waiting for scheduled async work to complete and then re-render (via `setTimeout`, `setState`, etc). 

https://legacy.reactjs.org/docs/test-utils.html#act